.env-template

```diff
@@ -0,0 +1 @@
+GATSBY_PERMITTED_METAMASK_NETWORK=
```

Comment:
why gatsby :)?

Reply:
GATSBY was removed and this is resolved.